autoreject/tests/test_ransac.py

```diff
@@ -36,16 +36,23 @@ def test_ransac():
     ransac = Ransac(picks=picks, random_state=np.random.RandomState(42))
     epochs_clean = ransac.fit_transform(epochs)
     assert len(epochs_clean) == len(epochs)
+
+    # Pass string instead of array of idx
+    picks = 'eeg'
+    ransac = Ransac(picks=picks, random_state=np.random.RandomState(42))
+    epochs_clean = ransac.fit_transform(epochs)
+    assert len(epochs_clean) == len(epochs)
```

Comment:
this is what one would call a smoke test. It's fine for a first pass but it doesn't explicitly test the `picks`. Can you check `ransac.picks` and assert that the channels are indeed all EEG channels (perhaps checked against output of `mne.pick_types`). Also, in order to keep the test fast you could `ransac.fit(epochs[:2])` or something like that

Reply:
Yes ok, this test was mostly just to check that it was not crashing. I'll add an assertion on a matching condition.

Reply:
Lucky enough I used `picks='eeg'`. If you set this test with `picks='mag'` and assert `ransac.picks` against `mne.pick_types(epochs.info, meg='mag', eeg=False, stim=False, eog=False, exclude=[])`, it matches. However, with EEG, one of the channels is dropped and ``ransac.picks` is missing one channel. Looking into that.

Reply:
Ok, I'm just a bit of an idiot. One channel was marked in `info['bads']`. This brings me to another problem: the default behavior of `_handle_picks(info, picks)` when `picks=None` was to exclude bads. Thus, on the change I made I also excluded bads. However in the test, and in my mind, it makes a lot more sense for autoreject, the `picks` are not excluding any channels. @jasmainak So is this exclusion of marked bad channels intended or is this a mistake? According to your reply, I will update accordingly `_handle_picks()` and the test with either an assertion against `picks = mne.pick_types(epochs.info, meg=False, eeg=True, stim=False, eog=False, exclude=[])` or `exclude='bads'`.

Reply:
Depending on this inclusion or exclusion of bad channels, I will update the different docstrings with something similar to MNE.

Reply:
I think that bad channels that were marked by the user should be ignored by Autoreject and Ransac.

Reply:
Ok, I'll make the update accordingly.